Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rudderanalytics"
-version = "1.0.0"
+version = "1.1.0"
 edition = "2018"
 readme = "README.md"
```

Comment:
`readme.md` file also needs to be updated. Let's remove the specific version and replace it with a placeholder.

Reply:
done